### PR TITLE
[FEAT] - create triangle and method variation

### PR DESCRIPTION
**Pull request by abosiy**

### CODE REVIEW

File: src/com/bosiy/Triangle.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+public class Triangle {
+    private final int sideA;
+    private final int sideB;
+    private final int sideC;
+
+    public Triangle(int sideA, int sideB, int sideC) {
+        this.sideA = sideA;
+        this.sideB = sideB;
+        this.sideC = sideC;
+    }
```
Comment (scratchw0w) [inline src/com/bosiy/Triangle.java:29]: formatting

File: src/com/bosiy/Triangle.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    public String triangle() {
+        return "side a = " + sideA + "; " + " side b = " + sideB + "; " +
+                " side c = " + sideC + "; ";
+    }
+    public String variation() {
```
Comment (scratchw0w) [inline src/com/bosiy/Triangle.java]: formatting

File: src/com/bosiy/Triangle.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        this.sideC = sideC;
+    }
+
+    @Override
+    public String toString() {
+        return "side a = " + sideA + "; " + " side b = " + sideB + "; " +
+                " side c = " + sideC + "; ";
+    }
+
+    public String variation() {
```
Comment (scratchw0w) [inline src/com/bosiy/Triangle.java]: rename method to verb.
all methods should be named as verbs.

for ex. getVariation, getTriangleType, getType etc.

File: src/com/bosiy/Triangle.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
+package com.bosiy;
+
+public class Triangle {
+    private final int sideA;
+    private final int sideB;
+    private final int sideC;
+
+    public Triangle(int sideA, int sideB, int sideC) {
```
Comment (scratchw0w) [inline src/com/bosiy/Triangle.java:8]: what if sides will be negative numbers?

Comment (scratchw0w) [inline src/com/bosiy/Triangle.java:8]: or what will be if I'll create unrealistic triangle?

for ex. first side = 1, second side = 1, and third side = 328

**Review by scratchw0w — CHANGES_REQUESTED** [review]

File: src/com/bosiy/Triangle.java (hunk 1 above)
Comment (abosiy) [reply src/com/bosiy/Triangle.java:29]: fixed

File: src/com/bosiy/Triangle.java (hunk 2 above)
Comment (abosiy) [reply src/com/bosiy/Triangle.java]: fixed

File: src/com/bosiy/Triangle.java (hunk 3 above)
Comment (abosiy) [reply src/com/bosiy/Triangle.java]: fixed

File: src/com/bosiy/Triangle.java (hunk 4 above)
Comment (abosiy) [reply src/com/bosiy/Triangle.java:8]: fixed

Comment (abosiy) [reply src/com/bosiy/Triangle.java:8]: fixed

**Review by scratchw0w — APPROVED** [review]

**Pull request merged**